example-streamer-uses/Cargo.toml

```diff
@@ -55,6 +55,6 @@ protobuf = { workspace = true }
 serde = { workspace = true }
 tokio = { workspace = true }
 up-rust = { workspace = true }
-up-transport-zenoh = { version = "0.1.1"  }
-up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.1.0", default-features = false }
-zenoh = { version = "1.0.0-alpha.6", features = ["unstable"]}
+up-transport-zenoh = { version = "0.3.0"  }
+up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.3.0", default-features = false }
+zenoh = { version = "1.0.0", features = ["unstable", "internal"] }
```

Comment:
FMPOV we should no longer depend on _unstable_ features and internal API. This only makes migration/updating to newer Zenoh versions harder ...

Reply:
Let me investigate if we need these still. We used to need.

Reply:
Okay! I was able to remove both "unstable" and "internal".